tests/links/37e91a9f-b04e-5313-bbc0-4c394406247e_2021-01-01T00:00:00.000000Z_90bf705c-d891-5ce2-aa54-094488b445c6.json

```diff
@@ -0,0 +1,80 @@
+{
+    "describedBy": "https://schema.staging.data.humancellatlas.org/system/3.0.0/links",
+    "links": [
+        {
+            "inputs": [
+                {
+                    "input_id": "af3549f4-8632-58c2-9821-c6249c622b0a",
+                    "input_type": "cell_suspension"
+                }
+            ],
+            "link_type": "process_link",
+            "outputs": [
+                {
+                    "output_id": "f0c8ac7c-1c4a-5c9c-9286-c8b16208f9ae",
+                    "output_type": "analysis_file"
+                }
+            ],
+            "process_id": "d7b8cbff-aee9-5a05-a4a1-d8f4e720aee7",
```

Comment:
A metadata file is missing from `schema-test-data/tests/metadata/process/` for this process ID.

Reply:
If this is resolved, please click

![image](https://user-images.githubusercontent.com/5143256/116593084-923e7800-a8d5-11eb-806b-2d1478211b6a.png)